library/core/game_test/src/main/java/org/quiltmc/qsl/game_test/api/package-info.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * <h2>The Game Test API.</h2>
+ *
+ * <p>
+ * <h3>What is the Game Test API?</h3>
+ * The Game Test API is the test automation framework used by Minecraft as most test frameworks are not adapted to a game that's so dynamic.
+ * Unit testing presents very quickly its issues: you will quickly find out that testing while in a tick loop is very difficult.
+ * Thus Mojang made their own testing framework, watching the <a href="https://www.youtube.com/watch?v=TNkPE6NTNHQ">keynote by Henrik Kniberg</a>
+ * is highly recommended to grasp the why, and how it works.
+ * <p>
+ * The Game Test API exposed by this module is a bit different, the framework has been adapted to work better with mods, with some extensions.
+ *
+ * <p>
+ * <h3>Get Started</h3>
+ * Writing tests for your mod is relatively simple: create a new class that extends {@link org.quiltmc.qsl.game_test.api.QuiltGameTest}, and add it
+ * as an entrypoint of key {@value org.quiltmc.qsl.game_test.api.QuiltGameTest#ENTRYPOINT_KEY}.
+ * <p>
+ * Now you can add public methods (may or may not be {@code static}) to the class,
+ * they take as argument {@link org.quiltmc.qsl.game_test.api.QuiltTestContext} which is an extension of {@link net.minecraft.test.TestContext},
+ * and annotating them with {@link net.minecraft.test.GameTest} will make them a test to run.
+ * <p>
+ * One of the special parts of game tests is they have a structure associated to them, as the goal is to test inside a game world with running
```

Comment:
hm, maybe this would be better?
```suggestion
 * One of the special parts of game tests is that they have a structure associated to them, as the goal is to test inside a game world with running
```